persistent_corpses.lua

```diff
@@ -165,6 +165,11 @@ if (SERVER) then
 		-- remove reference to the player so no more damage can be dealt
 		entity.ixPlayer = nil
 
+		-- remove player from netvars, we don't need to show any of the players data past death
+		entity:SetNetVar("player", nil)
+		-- we will show the characters data though
+		entity:SetNetVar("character", client:GetCharacter():GetID())
+
```

Comment:
You're still accessing the player via `GetPlayer` so there's no need to add this. Changing fairly core net vars like `player` should be avoided anyway due to potential breakages of other plugins/schemas that rely on it